inconsistent time conversions during daylight savings

<!--- Verify first that your issue is not already reported on GitHub -->
<!--- Also test if the latest release and devel branch are affected too -->
<!--- Complete *all* sections as described -->

#### Summary
When trying to run the chexus test suite, I stumbled upon an error seemingly unreleated to the code I changed:
`{'release_time': '2020-03-18T04:30:00'} != {'release_time': '2020-03-18T05:30:00'}`

When comparing the created TableItem time-related values,
```
        release_date="Feb. 1, 2020",
        release_time="12:30AM",
        release="On Feb. 1, 2020 at 12:30AM",
```
against the expected time-related values
```
        "release_date": "2020-02-01T05:00:00",
        "release_time": "%sT05:30:00" % date.today(),
        "release": "2020-02-01T05:30:00",
```
and the actual time-related values,
```
        "release_date": "2020-02-01T05:00:00",
        "release_time": "2020-03-18T04:30:00",
        "release": "2020-02-01T05:30:00",
```

only the release_time produces a discrepancy.

#### Issue Type
- Bug Report

#### Steps to reproduce
Run 'tox' during EDT.


#### Actual results
```
E       AssertionError: assert {'bad_datetim...omefile', ...} == {'bad_datetim...omefile', ...}
E         Omitting 9 identical items, use -vv to show
E         Differing items:
E         {'release_time': '2020-03-18T04:30:00'} != {'release_time': '2020-03-18T05:30:00'}
E         Full diff:
E           {
E            'bad_datetime': 'bats',
E            'file_name': 'somefile',...
E         
E         ...Full output truncated (13 lines hidden), use '-vv' to show

tests/test_models.py:48: AssertionError
=============================================================================== short test summary info ================================================================================
FAILED tests/test_models.py::test_table_item - AssertionError: assert {'bad_datetim...omefile', ...} == {'bad_datetim...omefile', ...}
```
Changing the release_time to one hour earlier temporarily fixes the problem.

#### Expected results
The test should return` "%sT05:30:00" % date.today()"`, not `"%sT04:30:00" % date.today()"`.



#### OS / Environment
<!--- Provide all relevant information, e.g. target OS, python and library version, etc. -->


#### Additional Information

The EDT/EST discrepancy when using "pytz.timezone("US/Eastern")" is noted here: http://pytz.sourceforge.net/#introduction
Apparently this can be fixed with the 'normalize' function.

--- a/chexus/_impl/models.py
+++ b/chexus/_impl/models.py
@@ -3,7 +3,6 @@ import json
 import os
 
 import dateutil
-import pytz
 
 
 class BucketItem(object):
@@ -67,10 +66,10 @@ class TableItem(object):
 
     Args:
         kwargs
-            Keyword arguments from which to create attributes.
+            Keyword arguments from which attributes are created.
             Dictionary values are converted to JSON strings.
             Values able to be parsed as dates and/or times are
-            converted to UTC timezone, ISO format datetime strings.
+            converted to ISO format datetime strings.
     """
 
     def __init__(self, **kwargs):
@@ -86,7 +85,7 @@ class TableItem(object):
 
     def _sanitize_value(self, value):
         # Coerce any dates, times to standard format
-        value = self._valid_datetime(value)
+        value = self._parse_datetime(value)
 
         # Serialize any dictionaries
         if isinstance(value, dict):
@@ -95,16 +94,9 @@ class TableItem(object):
         return value
 
     @staticmethod
-    def _valid_datetime(value):
+    def _parse_datetime(value):
         try:
-            # Parse string for datetime object
-            naive_dt = dateutil.parser.parse(value)
-            # Make datetime timezone-aware
-            aware_dt = pytz.timezone("US/Eastern").localize(naive_dt)
-            # Convert timezone to UTC
-            value = aware_dt.astimezone(pytz.utc).replace(tzinfo=None)
-            # Apply final formatting
-            return value.replace(tzinfo=None).isoformat()
+            return dateutil.parser.parse(value).isoformat()
         except (TypeError, ValueError):
             # String doesn't appear to be a date or time
             return value

--- a/tests/test_models.py
+++ b/tests/test_models.py
@@ -65,9 +65,9 @@ def test_table_item():
         "file_path": "path/to/somefile",
         "file_url": "www.example.com/content/path/to/somefile",
         "status": None,
-        "release_date": "2020-02-01T05:00:00",
-        "release_time": "%sT04:30:00" % date.today(),
-        "release": "2020-02-01T05:30:00",
+        "release_date": "2020-02-01T00:00:00",
+        "release_time": "%sT00:30:00" % date.today(),
+        "release": "2020-02-01T00:30:00",
         "bad_datetime": "bats",
         "metadata": '{"some": {"thing": [4, 5, 6]}}',
     }

